tasks/main.yml

```diff
@@ -1,10 +1,21 @@
 ---
 
+- name: Include OS-specific variables
+  include_vars: "{{ item }}"
+  with_first_found:
+    - "{{ ansible_distribution }}.yml"
+    - "{{ ansible_os_family }}.yml"
+
 - name: Include installation on Debian-based OS
   include: install.deb.yml
   when: ansible_os_family == 'Debian'
   tags: [mongodb]
 
+- name: Include installation on RedHat-based OS
```

Comment:
It's possible to combine with previous task:
include: "install.{{ ansible_os_family | lower }}.yml"